### PR TITLE
Side 1325 node self test loop

### PR DESCRIPTION
**Pull request by BrettMugglinOlive**

### CODE REVIEW

File: ldk/node/examples/selfTestLoop/package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "olive-helps-example-clipboard",
+  "name": "node-selft-test-loop",
```
Comment (bbrewer-oliveai) [inline ldk/node/examples/selfTestLoop/package.json:2]:
```suggestion
  "name": "node-self-test-loop",
```

File: ldk/node/examples/selfTestLoop/package.json [hunk 2]
```diff
@@ -29,6 +29,7 @@
   },
   "dependencies": {
     "@oliveai/ldk": "file:../..",
+    "moment": "^2.29.1",
```
Comment (bbrewer-oliveai) [inline ldk/node/examples/selfTestLoop/package.json:32]: It's worth noting that momentjs is "deprecated" and in maintenance mode. They do urge people to adopt a different util for this FYI.

File: ldk/node/examples/selfTestLoop/src/testingFixtures/testSuite.ts [hunk 3]
```diff
@@ -12,12 +17,56 @@ export default class TestSuite {
...
+      markdown: '',
+      elements,
+    });
+    const markdown = host.whisper.markdownWhisper({
+      label: 'Self Test - Results',
+      markdown: 'Press "Ctrl + /" to bring back up the original whisper',
+    });
+    const keyboard = host.keyboard.streamHotKey(hotkeys, (error, response) => {
+      if (error) {
+        // Do nothing
```
Comment (bbrewer-oliveai) [inline ldk/node/examples/selfTestLoop/src/testingFixtures/testSuite.ts:63]: Is there a reason we just toss the error?

**Review by bbrewer-oliveai — APPROVED** [review]
Looks pretty straightforward though most of the self-test stuff looked to be in place and this is adding on top?

Otherwise, is there a reason this is touching C# and Go? Is it this based off of Rich's branch?

File: ldk/node/examples/selfTestLoop/src/testingFixtures/testSuite.ts (hunk 3 above)
Comment (BrettMugglinOlive) [reply ldk/node/examples/selfTestLoop/src/testingFixtures/testSuite.ts:62]: This specific listener is to get back to the starting menu to pick which group you want to test. My thinking here being that if the tests works, this should work too. So why expect an error to be thrown?

File: ldk/node/examples/selfTestLoop/package.json (hunk 2 above)
Comment (BrettMugglinOlive) [reply ldk/node/examples/selfTestLoop/package.json:32]: I think I'm fine keeping it as moment for now, we can address it later

**Pull request merged**